src/util/utilities.ts

```diff
@@ -15,3 +25,18 @@ export function cleanListData(listData: ListData) {
 
   return cleanedListData
 }
+
+export function cleanBookData( bookData: BookData ) {
+  const cleanedBookData = bookData.map(book => {
+    return {
+      rank: book.rank,
+      publisher: book.publisher,
+      description: book.description,
+      title: book.title,
+      author: book.author,
+      bookImage: book.book_image,
+      amazonProductUrl: book.amazon_product_url
+    }
+  })
+  return cleanedBookData
```

Comment:
Good idea to include everything we might want - especially if we decide to work on displaying single book info pages.